package.json

```diff
@@ -101,6 +101,7 @@
     "marked": "1.2.3",
     "microtime": "^3.0.0",
     "mkdirp": "0.5.1",
+    "moment": "^2.29.1",
```

Comment:
@conartist6 Any thoughts about taking on devDependencies in order to test them? I think I would prefer not to, and since this bug already has a test from your PR, we don't need to write a new one.

Reply:
No I don't think we should if we can avoid it. Instead we should find the root cause of the incompatibility with such a dependency and simulate it ourselves. That way we make it clear to authors of any library what they can safely do. We don't want to discourage competition.

Reply:
Yeah, this was an issue that was fixed by https://github.com/immutable-js-oss/immutable-js/pull/17, so I'm just going to close and the corresponding issue since it's been fixed.